src/nreplController.ts

```diff
@@ -46,13 +46,19 @@ const start = (): Promise<CljConnectionInformation> => {
         });
 
         nreplProcess.stderr.on('data', data => {
+            console.info('nrepl stderr =>', data.toString());
```

Comment:
@mhansen Good idea. I'm gonna merge the PR as is but will create the related issue soon.